### PR TITLE
SOC-1276 Truncate usernames and fix author properties

### PR DESCRIPTION
**Pull request by mklucsarits**
These changes do two things:
1) fix missing/broken author properties in some components, and...
2) truncate long usernames so that no timestamp will wrap to a new line on small screens.

Related ticket: https://wikia-inc.atlassian.net/browse/SOC-1276

@garthwebb @rafalkalinski

### CODE REVIEW

File: front/templates/main/components/post-detail.hbs [hunk 1]
```diff
@@ -2,7 +2,8 @@
 	{{user-avatar
 		avatarUrl=author.avatarUrl
 		displayUsername=true
-		profileUrl=authorUrl
+		profileUrl=author.profileUrl
+		truncate=15
```
Comment (rogatty) [inline front/templates/main/components/post-detail.hbs]: `truncateName` maybe?

Comment (rafalkalinski) [inline front/templates/main/components/post-detail.hbs]: `truncateNameLength` even...

**Comment by rafalkalinski** [comment]
except variable name, LGTM

File: front/templates/main/components/post-detail.hbs (hunk 1 above)
Comment (rogatty) [inline front/templates/main/components/post-detail.hbs]: I leave specific naming in your hands but `user-avatar.hbs` currently expects `truncateName` not `truncate` ;)

Comment (mklucsarits) [inline front/templates/main/components/post-detail.hbs]: Whoops. I will also change it to ```truncateNameLength``` to make it more clear.

**Comment by garthwebb** [comment]
LGTM.  Btw, I ran the failing test again and it works now.  It seemed to be a failure in the build process, not with any specific with this branch.

File: front/styles/main/component/_user-avatar.scss [hunk 2]
```diff
@@ -2,6 +2,12 @@
 	display: inline-block;
 
 	.avatar-username {
+		display: inline-block;
 		font-size: $type-min;
+		max-width: 140px;
+		overflow: hidden;
+		text-overflow: ellipsis;
```
Comment (rogatty) [inline front/styles/main/component/_user-avatar.scss]: You can use `@extend %ellipsis`

**Pull request merged**